chaincfg/params.go

```diff
@@ -623,6 +626,94 @@ var TestNet3Params = Params{
 	SlpAddressPrefix:    "slptest",
 }
 
+var ChipNetParams = Params{
```

Comment:
Totally unnecessary and not desirable if the intent is to have clear static values for all the chain params but... if you want it to be more immediately understandable for the reader, you could move chipnet after testnet 4 and then reference almost all of the testnet 4 values as the chipnet values.

e.g. 
```go
BIP0034Height: 2,
// -->
BIP0034Height: TestNet4Params.BIP0034Height,
```

Reply:
I usually like things to be as explicit as possible but it's just opinion and not a hard one. Specially in this case.
So yeah I think it makes sense to change that. It will also be a bit more consistent with the rest of the code.